src/allmydata/test/test_humanreadable.py

```diff
@@ -0,0 +1,55 @@
+"""Tests for allmydata.util.humanreadable."""
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from past.builtins import long
+
+from twisted.trial import unittest
+
+from allmydata.util import humanreadable
+
+
+
+def foo(): pass # keep the line number constant
```

Comment:
is the point here to keep the line number constant across different python versions?  Maybe say that too (or the correct explanation if it's something else).  Or is this just advice to future maintainers not to put this in a different line?

Reply:
That was just copy/pasted, I'll clarify.